appengine/express/views/layout.jade

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015, Google, Inc.
+// Copyright 2016, Google, Inc.
```

Comment:
Not that this really matters but....  the way Copyrights are supposed to work, is that you include the range of years from the date it originally was released.  

  "the year of first publication of the work; in the case of compilations or derivative works incorporating previously published material, the year date of first publication of the compilation or derivative work is sufficient"